lightly/models/swav.py

```diff
@@ -0,0 +1,55 @@
+import torch
+from torch import nn
+
+from lightly.models.modules import SwaVProjectionHead, SwaVPrototypes
+from lightly.loss.memory_bank import MemoryBankModule
```

Comment:
We decided a while ago to not add "full" models to `lightly/models` anymore. The reason was that users usually want to customize models in some way or another and would have to change the source code to do so. Instead we started writing example model implementations in `examples/` that are easy to copy paste and modify.

Could you move everything in this file to `swav_new.py` (maybe rename the file to `swav_queue.py` or similar)? Then it would perfectly follow our examples "philosophy".